### PR TITLE
BV: Add support for sle12sp5

### PR DESCRIPTION
**Pull request by srbarrios**
## What does this PR change?

Add Cucumber features to support SLES12SP5 onboarding and test. This is a partial PR, the rest was made some time ago [here](https://github.com/uyuni-project/uyuni/pull/3338) 

**Depends on** : Support inside sumaform (https://github.com/uyuni-project/sumaform/pull/899)

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed

- [x] **DONE**

## Test coverage
- Cucumber tests were added

- [x] **DONE**

## Links

Ports:
- Manager-4.0
- Manager-4.1

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by srbarrios** [comment]
**_Checking now if we miss something on sumaform._**

File: testsuite/features/support/twopence_init.rb [hunk 1]
```diff
@@ -76,11 +76,11 @@ def twopence_init(target)
...
-  $client = $sle12sp4_client
-  $minion = $sle12sp4_minion
-  $ssh_minion = $sle12sp4_ssh_minion
-  $ceos_minion = $ceos7_ssh_minion
-  $ubuntu_minion = $ubuntu1804_minion
+  $client = $sle12sp5_client
+  $minion = $sle12sp5_minion
+  $ssh_minion = $sle12sp5_ssh_minion
+  $ceos_minion = $ceos8_ssh_minion
+  $ubuntu_minion = $ubuntu2004_minion
```
Comment (Bischoff) [inline testsuite/features/support/twopence_init.rb:83]: yes i would like to remove them for BV test suite. I will create a card.

**Review by Bischoff — APPROVED** [review]
just left a remark

**Review by Bischoff — CHANGES_REQUESTED** [review]
it's missing changes in `run_sets/build_validation_init_clients.yml`

**Comment by srbarrios** [comment]
> it's missing changes in `run_sets/build_validation_init_clients.yml`

Right!! Thanks!

**Review by Bischoff — APPROVED** [review]
thanks for the changes

**Pull request merged**